### ISSUE TITLE
I see no errors does not check if psalm ran successfully

### ISSUE THREAD

**Issue by mr-feek** [weirdan/codeception-psalm-module#6]
If the psalm process executed does not run successfully, the codeception test will still pass. 

Here is some output from a plugin test that I was developing, with `--debug`.

```
 When i run psalm
   I run psalm
  Moved to /Users/fiachra/repos/psalm-plugin-laravel/tests/_run
  Current version: 3.10.1
  Comparing 3.10.1.0 >= 3.4.0.0 => 1
  Running: /Users/fiachra/repos/psalm-plugin-laravel/vendor/vimeo/psalm/psalm --output-format=json  --no-progress
PHP Fatal error:  Cannot declare class Illuminate\Support\Collection, because the name is already in use in /Users/fiachra/repos/psalm-plugin-laravel/src/Stubs/Collection.stubphp on line 11

  Fatal error: Cannot declare class Illuminate\Support\Collection, because the name is already in use in /Users/fiachra/repos/psalm-plugin-laravel/src/Stubs/Collection.stubphp on line 11

  In Collection.stubphp line 11:

    Cannot declare class Illuminate\Support\Collection, because the name is already in use



  Moved to /Users/fiachra/repos/psalm-plugin-laravel
 Then i see no errors
   I can see no errors
 PASSED
```

I would expect this codeception test to fail. If the debug flag is not passed, then there is no indication that psalm did not run successfully.

**Comment by weirdan** [comment]
Indeed, I had this issue myself, but ignored it for the time being. Guess it's time to address it.

**Comment by mr-feek** [comment]
By the way, @weirdan thanks a lot for this! Nice and easy to  get started testing a plugin

**Comment by weirdan** [comment]
Fix released in [0.5.0](https://github.com/weirdan/codeception-psalm-module/releases/tag/0.5.0).
